cli/cobra.go

```diff
@@ -104,7 +104,7 @@ var usageTemplate = `Usage:
 {{- if not .HasSubCommands}}	{{.UseLine}}{{end}}
 {{- if .HasSubCommands}}	{{ .CommandPath}} COMMAND{{end}}
 
-{{ .Short | trim }}
+{{if ne .Long ""}}{{ .Long | trim }}{{ else }}{{ .Short | trim }}{{end}}
```

Comment:
Looks like this was changed in https://github.com/moby/moby/pull/23825/files#diff-4a2b6f090171fe18f7406cb69fb2e479R143

The original template was;

```
{{with or .Long .Short }}{{. | trim}}{{end}}{{if gt .Aliases 0}}
```

In a later PR, man-page generation was moved to external markdown files; https://github.com/moby/moby/pull/26830, but the template wasn't updated with that change, so I _think_ it's ok to use the long description again (if present)

ping @dnephin to double check

Reply:
Ya, seems fine to use Long now